### ISSUE TITLE
Error loading: snuids-trafficlights-panel

### ISSUE THREAD

**Issue by laurafernandezdiaz** [snuids/trafficlights-panel#36]
I have installed Grafana 8.2.1 and this plugin. However, I get this error when I try to use it:

![image](https://user-images.githubusercontent.com/51406942/196149020-4938ef75-aba9-4500-b217-a8f7deb69c8f.png)

I have already used this plugin before and I didn't have this problem. Do I have to do something special for this version?

**Comment by nba1992** [comment]
@laurafernandezdiaz  can you check your grafana logs?
i don't know if this is the same problem, but my logs are saying:
"
logger=plugin.loader t=2023-02-14T16:08:00.1714161Z level=warn msg="Skipping loading plugin due to problem with signature" pluginID=snuids-trafficlights-panel status=unsigned
"
I'm using the version 1.5.0.
According to the changelog, version 1.5.1 signed the plugin, but is not available for download. just the sources.zip.

**Comment by snuids** [comment]
You are right, I asked the submission a few months ago for version 1.5.1, but it was not accepted. I will try to fix that when I have spare time to waste.

**Comment by nba1992** [comment]
it is not a blocker, since you guys add the release tag, and explain, on documentation, how to install it using the dist foder. However, having the zip in the release is much easier. =D
